statvar_imports/us_bls/us_cpi/us_cpi_pvmap.csv

```diff
@@ -0,0 +1,59 @@
+key,p1,v1,p2,v2,p3,v3,,,,
+Place,observationAbout,{Data},populationType,ConsumerGoodsAndServices,measuredProperty,consumerPriceIndex,,,,
+#Relative importance,#ignore,#ignore,,,,,,,,
+Consumer Price Index,observationAbout,country/USA,,,,,,,,
+Unadjusted indexes,value,{Number},measurementQualifier,BLSSeasonallyUnadjusted,observationDate,{Year}-{Month},#Eval,"consumerGoodsCategory = str_to_camel_case(re.sub(""['\\-]"", """", Category.replace(' less ', ' except ')), '[^A-Za-z]')",#Aggregate,last
```

Comment:
Can the Eval cause new consumerGoodsCategory to be added which is missing in schema. If yes call that out in README